test/routes/appendix/appendix.test.js

```diff
@@ -115,6 +115,106 @@ describe('/appendix', () => {
     });
   });
 
+  describe('POST', () => {
+    test('/ - 200 Success with project/template combo', async () => {
+      const template1 = await db.models.template.create({name: uuidv4(),
```

Comment:
Theres a lot of repetitive setup/teardown, you could add a beforeEach/afterEach under the describe for these cases

Reply:
done!